examples/plotting/rgb_composite.py

```diff
@@ -0,0 +1,58 @@
+"""
+=============================
+Making an RGB composite image
+=============================
+
+This example shows the process required to create an RGB composite image
+of three AIA images at different wavelengths. To read more about the
+algorithm used in this example, see this
+`Astropy tutorial <https://docs.astropy.org/en/stable/visualization/rgb.html>`_.
+"""
+import matplotlib.pyplot as plt
+from matplotlib.lines import Line2D
+from sunkit_image.enhance import mgn
+
+from astropy.visualization import make_lupton_rgb
+
+import sunpy.data.sample
+from sunpy.map import Map
+
+###############################################################################
+# We will use three AIA images from the sample data at the following
+# wavelengths: 171, 193, and 211 Angstroms. The 171 image shows the quiet
+# solar corona, 193 shows a hotter region of the corona, and 211 shows
+# active magnetic regions in the corona.
+a171 = Map(sunpy.data.sample.AIA_171_IMAGE)
+a193 = Map(sunpy.data.sample.AIA_193_IMAGE)
+a211 = Map(sunpy.data.sample.AIA_211_IMAGE)
+
+maps = Map(a171, a193, a211)
```

Comment:
Can we not combine this?

If you pass in a list of sample files, it would create a sequence?